docs/tutorials/pre_executed/using_rubin_data.ipynb

```diff
@@ -0,0 +1,404 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.    # object_collection = lsdb.read_hats(base_path / "object_collection_lite")
</pre><p>open catalog instead of read=_hats</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/astronomy-commons/lsdb/pull/845/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>